code/modules/vending/clothesmate.dm

```diff
@@ -73,7 +73,7 @@
 		/obj/item/clothing/under/rank/captain/suit/skirt = 2,
 		/obj/item/clothing/under/rank/civilian/head_of_personnel/suit/skirt = 2,
 		/obj/item/clothing/suit/jacket = 2,
-		/obj/item/clothing/suit/hooded/wintercoat/custom = 2,
+		/obj/item/clothing/suit/hooded/wintercoat/custom = 5, //SKYRAT EDIT - INCREASED MARKET OR SOMETHING
```

Comment:
You could alternatively just remove it from there and keep it in the modular list.